index.html

```diff
@@ -11,7 +11,7 @@
     <title>Roscoe's Portfolio</title>
   </head>
   <body>
-    <header id="fixed_header">
+    <header class="fixed_header clearfix" id="fixed_header">
       <h1>Roscoe's Portfolio</h1>
     </header>
```

Comment:
Your navbar should be part of your header and it should be in a nav tag instead of in a div for better HTML code!